core/templates/dev/head/pages/error/error.html

```diff
@@ -4,52 +4,59 @@
 {% endblock navbar_breadcrumb %}
 
 {% block maintitle %}
-  {% if code == 400 %}
+  {% if status_code == 400 %}
     I18N_ERROR_PAGE_TITLE_400
-  {% elif code == 401 %}
+  {% elif status_code == 401 %}
     I18N_ERROR_PAGE_TITLE_401
-  {% elif code == 404 %}
+  {% elif status_code == 404 %}
     I18N_ERROR_PAGE_TITLE_404
-  {% elif code == 500 %}
+  {% elif status_code == 500 %}
     I18N_ERROR_PAGE_TITLE_500
   {% endif %}
 {% endblock maintitle %}
 
+
 {% block content %}
   <div class="oppia-content oppia-error-page-main-content" ng-controller="Error">
     <br>
 
     <div class="oppia-wide-panel oppia-error-wide-container">
       <div class="oppia-wide-panel-content protractor-test-error-container">
         <h1 class="oppia-error-h1">
-          {% if code == 400 %}
+          <div ng-if="errorCode === 400">
             <span translate="I18N_ERROR_HEADER_400"></span>
             - Bad Request
-          {% elif code == 401 %}
+          </div>
+          <div ng-if="errorCode === 401">
             <span translate="I18N_ERROR_HEADER_401"></span>
             - Unauthorized
-          {% elif code == 404 %}
+          </div>
+          <div ng-if="errorCode === 404">
             <span translate="I18N_ERROR_HEADER_404"></span>
             - Page Not Found
-          {% elif code == 500 %}
+          </div>
+          <div ng-if="errorCode === 500">
             <span translate="I18N_ERROR_HEADER_500"></span>
             - System Error
-          {% endif %}
+          </div>
         </h1>
 
         <br>
 
         <img ng-src="<[oopsMintImgUrl]>" alt="Oops!" width="299" height="142"/>
         <p>
-          {% if code == 400 %}
+          <div ng-if="errorCode === 400">
```

Comment:
Actually I think you can just do `<h2 ng-if="...">...</h2>` (without the div). Try it and see?